extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -263,9 +264,7 @@ export class AmpStoryPage extends AMP.BaseElement {
     this.mutator_ = Services.mutatorForDoc(getAmpdoc(this.win.document));
 
     /** @private @const {!../../../src/service/localization.LocalizationService} */
-    this.localizationService_ = Services.localizationServiceForDoc(
-      getAmpdoc(this.win.document)
-    );
+    this.localizationService_ = getLocalizationService(this.win.document.body);
```

Comment:
Passing `document.body` to get the `LocalizationService` here since there are cases where `amp-story-page` is not attached to the DOM yet, which prevents it from getting the service.